### ISSUE TITLE
A documentation fails to build in a sandbox

### ISSUE THREAD

**Issue by suhr** [mathics/Mathics#1070]
Command: `SANDBOX=true python mathics/test.py --doc-only`

```
builder for '/nix/store/frz27maw6nnrhj2fsvkgnarwc9rpgg7k-mathics-1.1.1.drv' failed with exit code 1; last 10 log lines:
      self.pattern.match(
    File "/nix/store/b6qpvqg5jss2ywm34x2iqhg0vl4i85kd-mathics-1.1.1/lib/python3.8/site-packages/mathics/core/pattern.py", line 137, in match
      yield_func(vars, None)
    File "/nix/store/b6qpvqg5jss2ywm34x2iqhg0vl4i85kd-mathics-1.1.1/lib/python3.8/site-packages/mathics/core/rules.py", line 40, in yield_match
      new_expression = self.do_replace(expression, vars, options, evaluation)
    File "/nix/store/b6qpvqg5jss2ywm34x2iqhg0vl4i85kd-mathics-1.1.1/lib/python3.8/site-packages/mathics/core/rules.py", line 129, in do_replace
      return self.function(evaluation=evaluation, **vars_noctx)
    File "/nix/store/b6qpvqg5jss2ywm34x2iqhg0vl4i85kd-mathics-1.1.1/lib/python3.8/site-packages/mathics/builtin/system.py", line 376, in evaluate
      return String(os.getlogin())
  OSError: [Errno 6] No such device or address
```

By the way, why does it try to evaluate tests even with `--doc-only`?

Mathics version: 1.1.1

**Comment by rocky** [comment]
@suhr you are the best person to find and fix such problems. Would you just make the changes that need changing? 

As to why it executes the test, I am guessing that in the doc it want's to show what the output woudl be.

**Comment by mmatera** [comment]
I think this was fixed in #1163. @suhr, could you confirm it?

**Comment by rocky** [comment]
Assuming this is fixed.
